src/server/main_service.cc

```diff
@@ -1576,19 +1576,21 @@ optional<bool> StartMultiEval(DbIndex dbid, CmdArgList keys, ScriptMgr::ScriptPa
   return false;
 }
 
-static std::string FullAclCommandFromArgs(CmdArgList args) {
+static std::string FullAclCommandFromArgs(CmdArgList args, std::string_view name) {
   ToUpper(&args[1]);
-  // Guranteed SSO no dynamic allocations here
-  return std::string("ACL ") + std::string(args[1].begin(), args[1].end());
+  auto res = absl::StrCat(name, " ", ArgS(args, 1));
+  return res;
 }
 
 std::pair<const CommandId*, CmdArgList> Service::FindCmd(CmdArgList args) const {
   const std::string_view command = facade::ToSV(args[0]);
-  if (command == "ACL") {
+  std::string_view acl = "ACL";
+  acl = registry_.RenamedOrOriginal(acl);
+  if (command == acl) {
```

Comment:
this is painful to read 😭 it's just
```
if (command == registry_cmd_.RenamedOrOriginal("ACL")) {}
```

(you can use command below because you checked it equals acl)